### PR TITLE
support getting retail customer data from pge, support cost in usage_summary and tou in interval reading. 

### PR DESCRIPTION
**Pull request by orlandoWexus**

### CODE REVIEW

**Comment by orlandoWexus** [comment]
@AndrewJo I made more changes to the gem to support getting customer information. Do you have time to take a look? Thanks

File: lib/green-button-data/parser/content.rb [hunk 1]
```diff
@@ -17,6 +17,8 @@ class Content
       element :ReadingType, class: ReadingType, as: :reading_type
       element :UsagePoint, class: UsagePoint, as: :usage_point
       element :UsageSummary, class: UsageSummary, as: :usage_summary
+      element :ServiceLocation, class: ServiceLocation, as: :service_location
+      element :CustomerAgreement, class: CustomerAgreement, as: :customer_agreement
```
Comment (AndrewJo) [inline lib/green-button-data/parser/content.rb]: Could you try to fit these under 80 character columns?

File: lib/green-button-data/parser/main_address.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+module GreenButtonData
+  module Parser
+    class MainAddress
+      include SAXMachine
+
+      def town_detail_info
+        town_detail.name + ',' + town_detail.code.gsub(/\s+/, "") + ',' + town_detail.state_or_province
```
Comment (AndrewJo) [inline lib/green-button-data/parser/main_address.rb]: Please refactor for 80 character limit.

**Comment by AndrewJo** [comment]
@orlandoWexus Just made few comments regarding code style but otherwise looks good! Let me know when you push with the changes and I'll merge this branch.

**Comment by orlandoWexus** [comment]
@AndrewJo done. thank you!

**Comment by orlandoWexus** [comment]
@AndrewJo just fixed broken test. Thanks again. :koala:

**Comment by AndrewJo** [comment]
Cool! Thanks for your contribution!

**Comment by AndrewJo** [comment]
Just released v0.6.0 to RubyGems.org!

**Pull request merged**